observability/README.md

```diff
@@ -44,8 +46,7 @@ sleep: 5
 -->
 
 ```bash
-cd ../hello-world
-dapr run --app-id hello-tracing --app-port 3000 node app.js
+cd ../hello-world && dapr run --app-id hello-tracing --app-port 3000 node app.js && cd -
```

Comment:
Does ```cd - ``` work in windows?

Reply:
No idea. Let me use the usual .. for this.